lib/github.rb

```diff
@@ -59,7 +59,12 @@ def commits_count
   # @return [Fixnum] the number of pull request comments
   def pull_request_comments_count
     events
-      .select { |event| event.type == 'PullRequestReviewCommentEvent' }
+      .select { |event|
```

Comment:
I did that first but thought it looks weird here. And I actually prefer using braces to indicate blocks that are used for their expression value, and do/end for more procedural blocks.